From f85bb8a398bb74d4b3206fc08f660153a78e380a Mon Sep 17 00:00:00 2001
From: lw312886 <lw312886@linux.alibaba.com>
Date: Wed, 2 Nov 2022 12:02:42 +0800
Subject: [PATCH 1/4] drivers: clk: modify clk gate bit for mipi csi1 cfg clock
  and mipi csi2 cfg clock

modify clk gate bit for mipi csi1 cfg clock  and mipi csi2 cfg clock

Signed-off-by: Wei Liu <wei.liu@linux.alibaba.com>
---
 drivers/clk/thead/gate/visys-gate.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/thead/gate/visys-gate.c b/drivers/clk/thead/gate/visys-gate.c
index 96a177561e6e..b023e42b8269 100644
--- a/drivers/clk/thead/gate/visys-gate.c
+++ b/drivers/clk/thead/gate/visys-gate.c
@@ -80,9 +80,9 @@ static int light_visys_clk_probe(struct platform_device *pdev)
 	gates[LIGHT_CLKGEN_MIPI_CSI0_CFG_CLK] = thead_gate_clk_register("clkgen_mipi_csi0_cfg_clk", NULL,
 									visys_regmap, 0xa0, 8, GATE_NOT_SHARED, NULL, dev);
 	gates[LIGHT_CLKGEN_MIPI_CSI1_CFG_CLK] = thead_gate_clk_register("clkgen_mipi_csi1_cfg_clk", NULL,
-									visys_regmap, 0xa0, 7, GATE_NOT_SHARED, NULL, dev);
-	gates[LIGHT_CLKGEN_MIPI_CSI2_CFG_CLK] = thead_gate_clk_register("clkgen_mipi_csi2_cfg_clk", NULL,
 									visys_regmap, 0xa0, 6, GATE_NOT_SHARED, NULL, dev);
+	gates[LIGHT_CLKGEN_MIPI_CSI2_CFG_CLK] = thead_gate_clk_register("clkgen_mipi_csi2_cfg_clk", NULL,
+									visys_regmap, 0xa0, 7, GATE_NOT_SHARED, NULL, dev);
 	gates[LIGHT_CLKGEN_DW200_CLK_VSE] = thead_gate_clk_register("clkgen_dw200_clk_vse", NULL,
 									visys_regmap, 0xa0, 5, GATE_NOT_SHARED, NULL, dev);
 	gates[LIGHT_CLKGEN_DW200_CLK_DWE] = thead_gate_clk_register("clkgen_dw200_clk_dwe", NULL,
-- 
2.17.1

